contracts/diamond/presets/OwnableDiamond/OwnableDiamond.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../../Diamond.sol";
+import "./OwnableDiamondStorage.sol";
+
+/**
+ *  @notice The Ownable preset of Diamond proxy
+ */
+contract OwnableDiamond is Diamond, OwnableDiamondStorage {
+    constructor() {
+        transferOwnership(msg.sender);
+    }
+
+    function transferOwnership(address newOwner) public onlyOwner {
```

Comment:
If we want to use such functions, we need to write something like `ProxyAdmin`. In current implementation all functions like `transferOwnership(), getOwner()...` won't be available in implementations of

Reply:
Yeah, this is ok